src/main/java/seedu/address/model/person/Customer.java

```diff
@@ -0,0 +1,100 @@
+package seedu.address.model.person;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Objects;
+import java.util.Set;
+
+import seedu.address.model.tag.Tag;
+import seedu.address.model.tag.UniqueTagList;
+
+/**
+ * Represents a Person in the address book.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class Customer extends Person {
+
+    private final Name name;
+    private final Phone phone;
+    private final Email email;
+    private final Address address;
+
+    private final UniqueTagList tags;
+
```

Comment:
Would declaration of these attributes be redundant, since they are already inherited from the `Person` parent class?